cmd/soroban-cli/src/config/network.rs

```diff
@@ -44,6 +53,17 @@ pub struct Args {
         help_heading = HEADING_RPC,
     )]
     pub rpc_url: Option<String>,
+    /// RPC Header(s) to include in requests to the RPC provider
+    #[arg(
+        long = "rpc-header",
```

Comment:
@elizabethengelman while trying out the pr locally I made this tweak, I renamed the option `--rpc-header` (no plural) so that at the command line it's more obvious that using the option is intended to be a single arg per use, so it looks like this:

```
stellar ... --rpc-header A:B --rpc-header C:D
```

So that folks don't think they can do:

```
stellar ... --rpc-headers A:B C:D
```

Reply:
sounds good, thanks for making that update!